presto-orc/src/test/java/com/facebook/presto/orc/metadata/statistics/TestMapColumnStatistics.java

```diff
@@ -42,20 +42,23 @@ public Object[][] keySupplier()
     @Test(dataProvider = "keySupplier")
     public void testEqualsHashCode(KeyInfo[] keys)
     {
-        MapColumnStatisticsBuilder builder1 = new MapColumnStatisticsBuilder();
+        MapColumnStatisticsBuilder builder1 = new MapColumnStatisticsBuilder(true);
```

Comment:
Changed this test to follow up on my mistake in the initial implementation of MapColumnStatisticsBuilder